release/tools/servlet.xml

```diff
@@ -22,7 +22,7 @@
 	<import file="../../bin/xml/ts.common.props.xml"/>
 
 	<property name="deliverable.version"  value="5.0"/>
-        <property name="deliverable.tck.version"  value="5.0.0"/>
+        <property name="deliverable.tck.version"  value="5.0.1"/>
```

Comment:
Please update to 5.0.2 for servlet.

We already released 5.0.1 via https://download.eclipse.org/jakartaee/servlet/5.0/jakarta-servlet-tck-5.0.1.zip.  Servlet + JAX-RS were the only TCK challenges.  

We started https://github.com/eclipse-ee4j/jakartaee-tck/wiki/TCK-Challenge-processing-status for tracking details of further TCK challenges.

We also started the https://github.com/eclipse-ee4j/jakartaee-tck/projects/2 project for tracking TCK challenges across EE release.

Reply:
Done.